.github/workflows/lock.yml

```diff
@@ -0,0 +1,20 @@
+name: 'Lock Threads'
+
+on:
+  schedule:
+    - cron: "0 */12 * * *"
+
+jobs:
+  lock:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/lock-threads@v2
+        with:
+          github-token: ${{ github.token }}
```

Comment:
shouldn't that be `${{ secrets.GITHUB_TOKEN }}`, like in [here](https://github.com/iv-org/invidious/blob/master/.github/workflows/stale.yml#L15)?

Reply:
Apparently not. Accordingly to the documentation on lock-threads the value must be `${{ github.token }}`. Maybe I've just interpreted it wrong but [take a look at it here if you're interested](https://github.com/marketplace/actions/lock-threads)

Reply:
Ah, right, my bad ^^